build.sbt

```diff
@@ -75,19 +82,20 @@ lazy val root = (project in file("."))
     `support-services`,
     `stripe-intent`,
     `support-redemptiondb`,
-    `it-test-runner`
+    `it-test-runner`,
+    `module-aws`,
+    `module-rest`,
```

Comment:
`rest` module was missing from the aggregate.  I tried removing the root project explicit definition, as SBT will auto aggregate all the projects for you, but it had a name clash with `support-frontend` subproject so it got an undesirable name.